setup.py

```diff
@@ -21,6 +21,7 @@ def read(fname):
     "magicgui >= 0.2.9",
     "aicsimageio ~= 4.0.5",
     "opencv-python-headless>=4.5.1",
+    "pyqt5"
```

Comment:
Should we put a version in here? looks like napari currently requires `PyQt5>=5.12.3,!=5.15.0` (https://github.com/napari/napari/blob/main/setup.cfg#L81)